### ISSUE TITLE
Game won't launch on Windows

### ISSUE THREAD

**Issue by Isakku** [ikemen-engine/Ikemen-GO#1086]
### Describe the bug

Doing it as described 

"Running

On windows, execute Ikemen_GO.exe (Ikemen_GO_x86.exe on 32-bit OS)"

I'm on Windows 8.1 64 bit. and  Ikemen_GO.exe doesn't do absolutely anything.
No error message, nothing!

Also there's no Ikemen_GO_x86.exe, that one worked in previous versions but it's not shipped in the latest 0.99 version!

I have been able to run the Linux version though.

### To Reproduce

1 Downloaded it from here https://github.com/ikemen-engine/Ikemen-GO/releases/tag/v0.99.0-rc.1
2 unzipped and saw Ikemen_GO_x86.exe was missing, only  Ikemen_GO.exe was available
3 Ikemen_GO_x86.exe worked fro me in the previous version whereas Ikemen_GO.exe doesn't work in either version (so the problem may be pre existent)
4 In my case, nothing happens, nothing launches, no error messages, nothing!

### Expected behavior

 Ikemen_GO.exe should run, my system is 64 bit Windows 8.1 with 12 Gb of RAM.
 
 Ikemen_GO_x86.exe should be available anyways

### Engine Version (or source code date)

v0.99.0-rc.1

### Operating system

Windows

**Comment by K4thos** [comment]
x86 are no longer supported. Maybe in future.

**Comment by Isakku** [comment]
> x86 builds are no longer supported. Maybe in future, if someone bothers to make a pull request for our [build pipeline](https://github.com/ikemen-engine/Ikemen-GO/blob/master/.github/workflows/releases.yml) with cross compilation (and test it), but it's not a priority.

Thanks for your reply. I have been eager to test this new version, but I was only able to run the Linux version.

Could you please help me help you? I would like to help solve issues like this one and maybe even add some features, but I'm not very used to Github nor the Go languaje. I had coded a few things in the past in Javascript, PHP, Visual Basic and C# but now I feel absolutely clueless on how to even get started here. What IDE should I use? Which tools should I get? I'm thinking about coding from a Linux machine as I don't think my old visual Studio will be useful for this here. 

Let's say I'd rather use FOSS tools if possible since those are easily available on Linux such as the Codeblocks IDE or Eclipse, but I need to learn the Go languaje as well. Could you please point out where I can get started? What are all the things I need to learn to get the code, build it, compile it, test it, and so on? Which IDE and what tools are you guys using?  Also, I think I will change from Manjaro (Rolling release) to something more stable like Open Suse Leap. (Manjaro has given me some headaches and it broke on my other machine) Can I work from my Linux machine or should I work on Windows instead? Maybe both?
Thanks in advance.

**Comment by EgyLynx** [comment]
Are it just that exe or some other file(s)? just open that exe and wait some seconds. it not open it immediatly but after some seconds.

**Comment by Isakku** [comment]
> Are it just that exe or some other file(s)? just open that exe and wait some seconds. it not open it immediatly but after some seconds.

Thanks for your reply. Nothing happens at all, if it were a matter of waiting a few seconds then that would been all, but nope, it just doesn't work at all.

**Comment by EgyLynx** [comment]
> > Are it just that exe or some other file(s)? just open that exe and wait some seconds. it not open it immediatly but after some seconds.
> 
> Thanks for your reply. Nothing happens at all, if it were a matter of waiting a few seconds then that would been all, but nope, it just doesn't work at all.
Open with Mouse´s second button?
AT my it open at easy .

**Comment by Isakku** [comment]
> Have you tried running the .exe using Command Prompt and seeing what comes up?

Thanks for your suggestion. Just tried that. Run CMD and then path tho Ikemen_Go.exe. Not much of a difference. All it spits out is a string that says Ikemen Go! and the date and the time, nothing else, no error messages, nothing that could give a hint of what could be wrong.

**Comment by assemblaj** [comment]
Hello @Isakku . 
If you don't mind, could you download and run this build and tell us what version of Open GL your computer supports, or just tell us outright if you know offhand? 

[Ikemen_GO-viewOpenGLVersion.zip](https://github.com/ikemen-engine/Ikemen-GO/files/14789307/Ikemen_GO-viewOpenGLVersion.zip)

Thanks

**Comment by Isakku** [comment]
Hi. @assemblaj  Sorry I didn't reply sooner. GPU-Z shows I have OpenGL 4.4.

**Comment by assemblaj** [comment]
> Hi. @assemblaj Sorry I didn't reply sooner. GPU-Z shows I have OpenGL 4.4.

Oh okay, I thought this might have been an Open GL version issue. Are you still having trouble opening IKEMEN?

**Comment by Isakku** [comment]
> > Hi. @assemblaj Sorry I didn't reply sooner. GPU-Z shows I have OpenGL 4.4.
> 
> Oh okay, I thought this might have been an Open GL version issue. Are you still having trouble opening IKEMEN?

Of course.  I think I remember testing with the latest version and still nothing changed, same problems.

Thanks for replying. I wish I could contribute more, but for now I'm busy, I need to reinstall one of my Manjaro machines.
